lib/Service/AttachmentService.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace OCA\Collectives\Service;
+
+use OCP\Files\File;
+use OCP\Files\Folder;
+use OCP\Files\InvalidPathException;
+use OCP\Files\NotFoundException as FilesNotFoundException;
+use OCP\IPreview;
+
+class AttachmentService {
+	private PageService $pageService;
+	private IPreview $preview;
+
+	public function __construct(PageService $pageService,
+								IPreview $preview) {
+		$this->pageService = $pageService;
+		$this->preview = $preview;
+	}
+
+	/**
+	 * @param File   $file
+	 * @param string $userId
+	 *
+	 * @return array
+	 * @throws NotFoundException
+	 */
+	private function fileToInfo(File $file, string $userId): array {
+		try {
+			return [
+				'id' => $file->getId(),
+				'name' => $file->getName(),
+				'filesize' => $file->getSize(),
+				'mimetype' => $file->getMimeType(),
+				'timestamp' => $file->getMTime(),
+				'path' => substr($file->getPath(), strlen('/' . $userId . '/files')),
```

Comment:
Not fully sure if this works for all cases, another option would be to use the user folder and call getRelativePath here, but fine with me if this approach also works.